daemon/algod/api/server/v2/dryrun.go

```diff
@@ -299,9 +296,9 @@ func (dl *dryrunLedger) LookupWithoutRewards(rnd basics.Round, addr basics.Addre
 			}
 		}
 	}
-	if !any {
-		return basics.AccountData{}, 0, fmt.Errorf("no account for addr %s", addr.String())
-	}
+	// Returns a 0 account for account that wasn't supplied.  This is new as of
```

Comment:
I don't remember what was the rationale here, but I do like that we're aligning the interface functionality. @pzbitskiy do you remember the details around this one ?